kimchi/src/verifier.rs

```diff
@@ -388,6 +388,19 @@ where
                     })
                     .collect::<Vec<_>>(),
             );
+            es.extend(
+                (0..COLUMNS)
```

Comment:
maybe we should have a `COEFFICIENTS`, since in theory we don't need one coefficient per register/column

Reply:
I agree. Lets open an issue for it, seems easy to pick up :)